src/checker.ml

```diff
@@ -965,6 +965,36 @@ let view_current_liquidation_auction_minimum_bid ((), state: unit * checker) : v
   ; minimum_bid = liquidation_auction_current_auction_minimum_bid auction
   }
 
+let view_current_liquidation_auction_id ((), state: unit * checker) : liquidation_auction_id option =
+  assert_checker_invariants state;
+  match state.liquidation_auctions.current_auction with
+  | None -> None
+  | Some current_auction -> Some current_auction.contents
```

Comment:
I think this one is part of `view_current_liquidation_auction_minimum_bid`, right?